uaclient/messages/__init__.py

```diff
@@ -1534,13 +1534,14 @@ def __repr__(self):
 the --access-only flag.""",
 )
 
-UNEXPECTED_ERROR = NamedMessage(
+UNEXPECTED_ERROR = FormattedNamedMessage(
     "unexpected-error",
     t.gettext(
         """\
-Unexpected error(s) occurred.
-For more details, see the log: /var/log/ubuntu-advantage.log
-To file a bug run: ubuntu-bug ubuntu-advantage-tools"""
+Unexpected error(s) occurred: {error_msg}
```

Comment:
While we're editing this - do we need the "(s)" after "error" here @lucasmoura @renanrodrigo? (Is there any chance we'll display more than one error?

Reply:
The only place I saw that used this error message and has multiple errors it reports is in contract.py. But it seems like it generates a separate error for each unexpected exception, instead of combining all the messages into one singular error?

Reply:
yeah we can use the singular I guess
If many errors occur, then one error occurred